src/main/java/core/basesyntax/write/WriteFile.java

```diff
@@ -1,12 +1,13 @@
 package core.basesyntax.write;
 
+import java.io.FileWriter;
 import java.io.IOException;
 
-public class WriteFile implements FileWriter {
+public class WriteFile implements FileWriterServ {
```

Comment:
```suggestion
public class FileWriterImpl implements FileWriter {
```
let's be consecutive in our namings
if we already have FileReader interface and FileReaderImpl implementing FileReader